gnuplot/plan.sh

```diff
@@ -0,0 +1,47 @@
+pkg_name=gnuplot
+pkg_origin=core
+pkg_version="5.0.6"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('gnuplot')
+pkg_source="https://sourceforge.net/projects/$pkg_name/files/$pkg_name/$pkg_version/$pkg_name-$pkg_version.tar.gz"
+pkg_shasum="5bbe4713e555c2e103b7d4ffd45fca69551fff09cf5c3f9cb17428aaacc9b460"
+pkg_deps=(
```

Comment:
The list is already long, but gnuplot links to these other dependencies (this is needed for proper rdeps rebuild):

* `core/bzip2`
* `core/jbigkit`
* `core/libffi`
* `core/libice`
* `core/libiconv`
* `core/libjpeg-turbo`
* `core/libsm`
* `core/libtiff`
* `core/libxau`
* `core/libxcb`
* `core/libxdmcp`
* `core/libxext`
* `core/ncurses`
* `core/xlib`
* `core/xz`